src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Lottery getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier supplier = new ColorSupplier();
+        ball.color = supplier.getRandomColor();
+        ball.number = new Random().nextInt(100);

```

Comment:
same lets make Random and ColorSupplier as class level variables, not to create new instances every time we call getRandomBall() method. We need to use setters or constructor to initialize Ball fields. Lets move 101 to constant https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names